docs/getting_started.rst

```diff
@@ -202,238 +203,181 @@ Alternatively, you can run the devserver with `hot reload <https://vue-loader.vu
 
+.. code-block:: bash
 
+  yarn run frontend-devserver
 
 
-Developing on Kolibri inside Docker
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Editor configuration
+--------------------
 
-.. note::
-  The Docker workflows below have not been fully tested
+We have a project-level *.editorconfig* file to help you configure your text editor or IDE to use our internal conventions.
 
-Users who are familiar with Docker can spin up a Kolibri instance quickly without setting up
-the full JavaScript and Python development environments. We provide docker images that contain
-all the necessary prerequisites for running Kolibri.
+`Check your editor <http://editorconfig.org/#download>`__ to see if it supports EditorConfig out-of-the-box, or if a plugin is available.
 
-The ``docker/`` directory contains the docker files and startup scripts needed for various tasks.
 
- * ``docker/base.dockerfile``: the base layer that installs JavaScript and Python dependencies (image tag ``learningequality/kolibri``).
- * ``docker/build_whl.dockerfile``: generates a ``.whl``, ``tar.gz``, and ``.pex`` files in ``dist/``
- * ``docker/build_windows.dockerfile``: used to generate the Windows installer.
- * ``docker/dev.dockerfile``: container with full development setup, running devserver.
- * ``docker/demoserver.dockerfile``: runs the pex from ``KOLIBRI_PEX_URL`` with production setup.
- * ``docker/entrypoint.py``: startup script that configures Kolibri based on ENV variables:
+Vue development tools
+---------------------
 
-    * Set ``KOLIBRI_PEX_URL`` to string ``default`` to run latest pex from Kolibri download page
-    * Set ``KOLIBRI_PEX_URL`` to something like ``http://host.org/nameof.pex``
-    * Set ``DOCKERMNT_PEX_PATH`` to something like ``/docker/mnt/nameof.pex``
-    * ``KOLIBRI_RUN_MODE``: set in Dockerfile
-    * ``KOLIBRI_PROVISIONDEVICE_FACILITY``: if this environment variable is set
-      the entrypoint script will run the provision device an setup a facility
-      with this name. The ``KOLIBRI_LANG`` environment variable and the following
-      other environment variables will be used in the process:
+`Vue.js devtools <https://github.com/vuejs/vue-devtools>`__ is a browser plugin that is very helpful when working with Vue.js components and Vuex.
 
-        * ``KOLIBRI_PROVISIONDEVICE_PRESET``: defaults to ``formal``, with the other options being ``nonformal`` and ``informal``
-        * ``KOLIBRI_PROVISIONDEVICE_SUPERUSERNAME``: default ``devowner``
-        * ``KOLIBRI_PROVISIONDEVICE_SUPERUSERPASSWORD``: default ``admin123``
+To ensure a more efficient workflow, install appropriate editor plugins for Vue.js, ESLint, and stylelint.
 
-    * ``KOLIBRI_HOME``: default ``/kolibrihome``
-    * ``KOLIBRI_HTTP_PORT``: default ``8080``
-    * ``KOLIBRI_LANG``: default ``en``
-    * ``CHANNELS_TO_IMPORT``: comma-separated list of channel IDs (not set by default)
 
+Sample data for development
```

Comment:
Suggest changing this to `Import channels and resources for development`.
`Data` may suggest that what is being imported is interaction- and DB-like _data_.

Reply:
The diff makes it confusing to see, but this section now also includes:

> Now you can create users, classes, lessons, etc manually. To auto-generate some sample user data you can also run:
> 
> ```bash
>    kolibri manage generateuserdata
> ```

Reply:
Fair enough, could you then change the heading to `Sample resources and data for development` to make it more comprehensive? Thank you!

Reply:
updated!